TestHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -745,11 +760,16 @@ public void MockDirectory_GetFiles_ShouldThrowAnArgumentException_IfSearchPatter
 
         public static IEnumerable<string> GetSearchPatternForTwoDotsExceptions()
         {
-            yield return @"a..\b";
             yield return @"a../b";
             yield return @"../";
-            yield return @"..\";
-            yield return @"aaa\vv..\";
+
+            if (!MockUnixSupport.IsUnixPlatform()) 
```

Comment:
This is a little confusing to me. If we're not on a Unix platform, go ahead and add these values to the test data.. but the comment says that these aren't a problem on Unix, so why would we exclude these? If they do need to be excluded, I would go with a similar approach as outlined above where we'd just go ahead and remove these and leverage a Windows test.